homeassistant/util/dt.py

```diff
@@ -41,6 +41,18 @@ def utcnow():
     return dt.datetime.now(UTC)
 
 
+def beatsnow():
```

Comment:
Yes. Beats are not common to be used and so I don't want it to be in memory
for 100% users while <1% will use it.

On Wed, May 25, 2016, 19:48 Paul Philippov <notifications@github.com> wrote:

> In homeassistant/util/dt.py
> <https://github.com/home-assistant/home-assistant/pull/2167#discussion_r64685094>
> :
>
> > @@ -41,6 +41,18 @@ def utcnow():
> >      return dt.datetime.now(UTC)
> >
> >
> > +def beatsnow():
>
> What do you mean common enough? Basically, it's an extension of datetime
> module to report time in .beats. Why should a class level function reside
> in a component code? What if I want to reuse .beats for another component,
> or a script? Should I copy and paste that function all over the place then?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/home-assistant/home-assistant/pull/2167/files/1e350cef9802b919d5f446fbe47667045ed75aba#r64685094>
>